db/R/utils.R

```diff
@@ -317,22 +326,26 @@ db.getShowQueries <- function() {
 ##' @author David LeBauer
 ##' @examples
 ##' \dontrun{
-##' pftid <- get.id("pfts", "name", "salix", con)
-##' pftid <- get.id("pfts", c("name", "modeltype_id"), c("ebifarm.salix", 1), con)
+##' pftid <- get.id('pfts', 'name', 'salix', con)
+##' pftid <- get.id('pfts', c('name', 'modeltype_id'), c('ebifarm.salix', 1), con)
 ##' }
-get.id <- function(table, colnames, values, con, create=FALSE, dates=FALSE){
+get.id <- function(table, colnames, values, con, create = FALSE, dates = FALSE) {
```

Comment:
Second argument here is colnames (plural). In some of the calls to this function we're using colname, others are using colnames, and others aren't passing by name. Need to make this consistent. colname is clearly wrong.